packages/vite/src/node/config.ts

```diff
@@ -963,6 +965,8 @@ export async function loadConfigFromFile(
   configFile?: string,
   configRoot: string = process.cwd(),
   logLevel?: LogLevel,
+  customLogger?: Logger,
+  allowClearScreen?: boolean,
```

Comment:
I don't think we need to pass `allowClearScreen` here. The other `createLogger` calls, especially in `cli.ts` didn't pass this too. If anything, maybe we should disable clearScreen for special loggings like this, but that could be a different issue.